### PR TITLE
Upgrade yeoman-generator to 1.1.1 🎉

### LINKED ISSUE #80: Migrate to Yeoman 1.0
http://yeoman.io/blog/hello-generator-1.0.html

### PR DESCRIPTION
**Pull request by kutyel**
And in the way the rest of the dependencies *(its free)* 😜 

Closes #80

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ node_modules/
 temp/
 npm-debug.log
 .DS_Store
+.vscode
```
Comment (Kikobeats) [inline .gitignore]: 😏

**Comment by Kikobeats** [comment]
The magic question is: are you tested it? 😄

**Comment by Kikobeats** [comment]
btw

File: .gitignore (hunk 1 above)
Comment (kutyel) [inline .gitignore]: He visto el `.idea` por ahí en otros proyectos tuyos así que no te quejes 😝

**Comment by kutyel** [comment]
Yep! I have tested it with Node 4.8.x and works like charm! ✨

**Comment by Kikobeats** [comment]
don't say more 😂

**Comment by kutyel** [comment]
🤣 🤣 🤣  finally the `devDepedencies` badge will be green again! 🎉 🎉 🎉

**Pull request merged**